cluster/config.go

```diff
@@ -44,7 +44,11 @@ func getK8sAPIServer() string {
 
 // Returns the namespace in which the controller is installed
 func GetNs() string {
-	return "default"
+	dat, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/namespace")
```

Comment:
looks ugly but it's the most accepted answer in stackoverflow and github :p, LGTM